### ISSUE TITLE
All hyphens stripped from ISBNs

### ISSUE THREAD

**Issue by logological** [plk/biber#89]
When using the new Biber 2.2 and biblatex 3.1, all hyphens are now stripped from ISBNs.  This happens even when passing the `--isbn-normalise` option to Biber.  Minimal working example:
````
\documentclass{article}
\usepackage[backend=biber]{biblatex}
\addbibresource{bibliography.bib}
\begin{filecontents}{bibliography.bib}
@book{		  iverson1962programming,
  Author       = {Kenneth E. Iverson},
  Title	       = {A Programming Language},
  Year	       = 1962,
  Publisher    = {Wiley},
  isbn	       = {978-0-471-43014-8},
  Address      = {New York},
}
\end{filecontents}
\begin{document}
\nocite{*}\printbibliography
\end{document}
````
With or without `--isbn-normalise`, this outputs the following:
````
References
[1] Kenneth E. Iverson. A Programming Language. New York: Wiley, 1962. ISBN: 9780471430148.
````
But I was expecting the following:
````
References
[1] Kenneth E. Iverson. A Programming Language. New York: Wiley, 1962. ISBN: 978-0-471-43014-8.
````

**Comment by plk** [comment]
This was a typo in the code. Please try biber version 2.3 from Sourceforge (you will also need biblatex 3.2 from Sourceforge).

**Comment by logological** [comment]
Yes, the problem no longer occurs with biber 2.3 + biblatex 3.2.

**Comment by surmann** [comment]
Thanks for the fix, when will the couple (biber 2.3 and biblatex 3.2) be released on CTAN?

**Comment by plk** [comment]
I will try to get the releases done this week.
